### DISCUSSION TITLE
на 1-2 секунды появляется стандартный дизайн панели громкости

### DISCUSSION THREAD

**Discussion by Derbrent** [ShankarBUS/ModernFlyouts#907] [General]
После загрузки системы ОС Windows 10, и при использовании ползунка громкости, на 1-2 секунды появляется стандартный дизайн панели громкости. Можно ли это исправить ?

**Comment by Samuel12321** [comment]
Translation:

the standard volume bar design appears for 1-2 seconds
After booting into Windows 10, and using the volume slider, the standard volume bar design appears for 1-2 seconds. Can this be fixed ?

Answer:
We are working on this, there is a bug open for this problem.
